src/Poseidon/CLI/Forge.hs

```diff
@@ -48,15 +50,17 @@ import           System.IO                   (hPutStrLn, stderr)
 -- | A datatype representing command line options for the survey command
 data ForgeOptions = ForgeOptions
     { _forgeBaseDirs     :: [FilePath]
+    , _forgeInGenos      :: [InGenotypeData]
     , _forgeEntitySpec   :: Either SignedEntitiesList FilePath
+    , _forgeSnpFile      :: Maybe FilePath
     , _forgeIntersect    :: Bool
-    , _forgeOutPacPath   :: FilePath
-    , _forgeOutPacName   :: Maybe String
     , _forgeOutFormat    :: GenotypeFormatSpec
     , _forgeOutMinimal   :: Bool
+    , _forgeOutOnlyGeno  :: Bool
```

Comment:
As I know you like types reflecting program correctness: Here you could tie `onlyGeno` with `outName` in a clever way. Since we know that only package-output will require a name, you could define a datatype `data ForgeOutputSpec = ForgeOutputPackage String | ForgeOutputOnlyGeno`, which then forces the CLI to ensure that you can't do something like `--onlyGeno --pacName XXX`.

Reply:
I appreciate the idea, but as the connection between `onlyGeno` and `pacName` is a bit indirect, I think I prefer too keep things simple here. A `pacPath` is necessary anyway and the name is just a `Maybe`...